src/components/NewsCarouselItem/NewsCarouselItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { Grid } from '@trussworks/react-uswds'
+import React from 'react'
+import styles from './NewsCarouselItem.module.scss'
+import { ArticleListItemRecord } from 'types'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+const NewsCarouselItem = ({ article }: { article: ArticleListItemRecord }) => {
+  return (
+    <Grid row>
+      <Grid
+        tablet={{ col: 'fill' }}
+        desktop={{ col: 'auto' }}
+        className={styles.imageContainer}>
+        {/* TODO: current image is a placeholder and will need to be updated */}
```

Comment:
Question: Is this pending the S3 image upload work or is this something else?

Reply:
Correct. I just grabbed the image from a recent article in the RSS feed.